### PR TITLE
Wip swe 1272 content cover screen

### PR DESCRIPTION
**Pull request by Anhelina4**
image

### CODE REVIEW

File: src/components/elements/Typography/Title/Title.js [hunk 1]
```diff
@@ -14,12 +14,14 @@ const DEFAULT_TAG = 'h1'
...
 const Title = (props) => {
-  const { as, center, secondary, children, className, ...rest } = props
+  const { as, center, secondary, children, className, color, style, ...rest } =
+    props
 
   // [COMPUTED PROPPERTIES]
   const tag = as || DEFAULT_TAG
 
   return createElement(tag, {
+    style: { style, color, ...rest },
```
[the remark this answers is not in the document]
Comment (Anhelina4) [reply src/components/elements/Typography/Title/Title.js:24]: fixed

File: src/components/elements/Typography/Title/Title.js [hunk 2]
```diff
@@ -14,12 +14,14 @@ const DEFAULT_TAG = 'h1'
  * prop, or `h1` if no `as` prop is provided. The component's class name is set to the tag name.
  */
 const Title = (props) => {
-  const { as, center, secondary, children, className, ...rest } = props
+  const { as, center, secondary, children, className, color, style, ...rest } =
```
Comment (5P4C3GR33D) [inline src/components/elements/Typography/Title/Title.js:17]: Check propTypes.

File: src/screens/ContentCoverScreen/ContentCoverScreen.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+  // [HELPER FUNCTION]
+  const titleToUpperCase = (string) => {
+    return string.toUpperCase()
+  }
+
+  return (
+    <ContentCoverScreenWrapper>
+      <div className="container-fluid d-flex flex-column-reverse height-inherit">
+        {content?.reverse().map((item) => (
+          <div className="row justify-content-end">
```
Comment (5P4C3GR33D) [inline src/screens/ContentCoverScreen/ContentCoverScreen.js:30]: It seems that this row/col are excess. Check this, please.

File: src/components/elements/Typography/Title/Title.js (hunk 2 above)
Comment (Anhelina4) [reply src/components/elements/Typography/Title/Title.js:17]: fixed

**Review by 5P4C3GR33D — APPROVED** [review]

**Pull request closed without merging**